deployment/distribution/build.gradle.kts

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+// Build file
+apply(plugin = "application")
+
+description = "Script and CLI to run the tests."
+
+group = "org.codice.samlconf.deployment"
+
+configure<ApplicationPluginConvention> {
+    applicationName = "samlconf"
+    mainClassName = "org.codice.ctk.CommandKt"
+
+    applicationDefaultJvmArgs = listOf("-Dapp.home=SAMLCTK_APP_HOME")
+
+    applicationDistribution.from("src/main/resources/") {
+        into( "conf")
+    }
+
+    // Copy implementation examples into the distribution
+    project(":external:implementations").getSubprojects().forEach { subProject ->
+        applicationDistribution
+                .into("implementations/" +
+                        subProject.name.replace("samlconf-", "")
+                                .replace("-impl", "")) {
+            from (subProject.file("build/resources/main"))
+            from (subProject.file("build/libs"))
+        }
+    }
+}
+
+dependencies {
+    compile(project(":external:implementations:samlconf-ddf-impl"))
+    compile(project(":external:implementations:samlconf-keycloak-impl"))
+    compile(project(":library"))
+    compile(project(":external:samlconf-plugins-api"))
+    compile(project(":ctk:common"))
+    compile(project(":ctk:idp"))
+
+    compile("us.jimschubert:kopper-typed:0.0.3")
+    compile("org.fusesource.jansi:jansi:1.17.1")
```

Comment:
❓ Do we want to centralize all dependency versions into the `Versions` object, even single usages like this?

Reply:
Certainly can. I hadn't done the one-offs, but it's not a big deal to do so.